pkg/internal/apis/certmanager/types_issuer.go

```diff
@@ -177,6 +177,9 @@ type VaultIssuer struct {
 	// "my_pki_mount/sign/my-role-name".
 	Path string
 
+	// Namespace is the name of the vault namespace
```

Comment:
Can you expand the documentation on this to explain what a vault namespace is (a link to the Vault docs would do it), an example usage, as well as explanation of the behaviour of it is not specified and when it *is* required?

Reply:
Thanks @munnerz, tried to keep it as short as possible, but let me know if more needs to be added 👍